dev-haskell/ghc-api-compat/ghc-api-compat-8.6.1_pre20210712.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+# The version on hackage isn't properly packaged, so we use the live version
+# for now
+inherit haskell-cabal git-r3
```

Comment:
Instead of relying on live `git` repository you can refer to a tarball, say: https://gitlab.haskell.org/haskell/ghc-api-compat/-/archive/1c4b23e8f929b052f2172ca509c7428811547b89/ghc-api-compat-1c4b23e8f929b052f2172ca509c7428811547b89.tar.bz2

Reply:
Good idea